core/build/src/lib.rs

```diff
@@ -44,6 +44,7 @@ pub fn sgx_builder() -> Builder {
         .use_core()
         .ctypes_prefix("core::ffi")
         .allowlist_recursively(false)
+        .parse_callbacks(Box::new(SgxParseCallbacks))
```

Comment:
This showed up in at least one other PR I reviewed today - I assume it'll all shake out as we collapse the PR tree, we just need to make sure it doesn't end up getting merged twice.

Reply:
yes I'm hoping it doesn't collide, I wanted this change and the rename change which happened to be parallel branches